### PR TITLE
Rockerbox updates

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Describe the changes you are proposing in this PR.

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Review by yashasvibajpai — APPROVED** [review]
Looks good!

File: docs/destinations/streaming-destinations/rockerbox/setting-up-rockerbox.mdx [hunk 1]
```diff
@@ -23,32 +23,20 @@ Once you have confirmed that the source platform supports sending events to Rock
 
 To successfully configure Rockerbox as a destination, configure the following settings:
 
-<img src="../../../assets/event-stream-destinations/rockerbox-connection-settings.png" alt="Rockerbox Connection Settings in RudderStack" />
+- **Advertiser Id**: Enter your Rockerbox Advertiser Id. It is a required field for sending events to Rockerbox via <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>. To obtain it, contact your <a href="https://help.rockerbox.com/article/5t050dmcxv-webhooks#requirements:~:text=or%20other%20formats-,Server%2Dside%20specs,-You%20will%20need">Rockerbox account manager</a>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/rockerbox/setting-up-rockerbox.mdx:26]:
```suggestion
- **Advertiser Id**: Enter your Rockerbox Advertiser ID. It is a required field for sending events to Rockerbox via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>. To obtain it, contact your <a href="https://help.rockerbox.com/article/5t050dmcxv-webhooks#requirements:~:text=or%20other%20formats-,Server%2Dside%20specs,-You%20will%20need">Rockerbox account manager</a>.
```

File: docs/destinations/streaming-destinations/rockerbox/setting-up-rockerbox.mdx [hunk 2]
```diff
@@ -23,32 +23,20 @@ Once you have confirmed that the source platform supports sending events to Rock
...
 
-- **Advertiser Id**: Enter your Rockerbox Advertiser Id.
+- **Map your Rockerbox events to RudderStack Events**: Enter the corresponding RudderStack and Rockerbox events you want to be mapped. You can specify multiple <strong>RudderStack events</strong> for one <strong>Rockerbox event</strong> and vice versa.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/rockerbox/setting-up-rockerbox.mdx:28]:
```suggestion
- **Map your Rockerbox events to RudderStack Events**: Use this setting to map your RudderStack events to specific Rockerbox events. You can map multiple <strong>RudderStack events</strong> to one <strong>Rockerbox event</strong> and vice versa.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/rockerbox/setting-up-rockerbox.mdx:28]: LMK if this suggestion makes sense.

File: docs/destinations/streaming-destinations/rockerbox/setting-up-rockerbox.mdx [hunk 3]
```diff
@@ -23,32 +23,20 @@ Once you have confirmed that the source platform supports sending events to Rock
...
-<div class="infoBlock">
-Advertiser Id is a required field for sending events to Rockerbox via <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>. To obtain it, contact your <a href="https://help.rockerbox.com/article/5t050dmcxv-webhooks#requirements:~:text=or%20other%20formats-,Server%2Dside%20specs,-You%20will%20need">Rockerbox account manager</a>.
-</div>
-
-- **Map your Rockerbox events to RudderStack Events**: Enter the corresponding RudderStack and Rockerbox events you want to be mapped.
-
-<div class="successBlock">
-You can specify multiple <strong>RudderStack events</strong> for one <strong>Rockerbox event</strong> and vice versa.
-</div>
+- **Map your event property to Rockerbox custom property**: Enter the RudderStack event property and a corresponding Rockerbox property you want to send to Rockerbox as a custom property. To send these custom properties to Rockerbox, you must send them in the `track` call.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/rockerbox/setting-up-rockerbox.mdx:30]:
```suggestion
- **Map your event property to Rockerbox custom property**: Use this setting to map your RudderStack event properties to specific Rockerbox properties. RudderStack sends these fields to Rockerbox as custom properties. To send these custom properties to Rockerbox, you must send them in the `track` call.
```

**Pull request merged**